Gemfile

```diff
@@ -14,13 +14,14 @@ group :test do
   gem 'rake'
   gem 'puppet', ENV['PUPPET_GEM_VERSION'] || '~> 4'
   gem 'facter', '>= 2.0'
-  gem 'rspec-puppet', :git => 'https://github.com/rodjek/rspec-puppet.git'
+  gem 'rspec-puppet', git: 'https://github.com/rodjek/rspec-puppet.git'
   gem 'puppetlabs_spec_helper', '< 1.0'
   gem 'metadata-json-lint'
   # json_pure 2.0.2 added a requirement on ruby >= 2. We pin to json_pure 2.0.1
   # if using ruby 1.x
-  gem 'json_pure', '<=2.0.1', :require => false if RUBY_VERSION =~ /^1\./
+  gem 'json_pure', '<=2.0.1', require: false if RUBY_VERSION =~ /^1\./
   gem 'rubocop', '~> 0.41.0'
+  gem 'rubocop-rspec'
```

Comment:
In lieu of modulesyncing, please see https://github.com/puppetlabs/modulesync_configs/blob/master/moduleroot/Gemfile#L48-L49 for the recommended magic incantation.